code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -329,7 +329,7 @@
 		playsound(loc, 'sound/effects/glass_step.ogg', 50, 1)
 		if(ishuman(AM))
 			var/mob/living/carbon/human/H = AM
-			if(!(PIERCEIMMUNE in H.dna.species.specflags))
+			if(!(PIERCEIMMUNE in H.dna.species.specflags) || !istype(buckled_mob,/obj/structure/bed/chair/withwheels))
```

Comment:
The issue I have with this is it only affects chairs with wheels and might not encompass all wheeled furniture you can sit/lie on. For example there the roller bed:

https://github.com/HippieStationCode/HippieStation13/blob/5462010e5bcc8a61ad5618ffd0f04f07eeb570b0/code/game/objects/structures/stool_bed_chair_nest/bed.dm#L84

If we have to add special cases each time then it gets kinda snowflakey. What if we checked the item H is buckled to has wheels using a new var on the wheeled furniture?